helios-ts/package.json

```
@@ -1,22 +1,62 @@
 {
   "name": "helios",
   "version": "0.1.0",
-  "main": "./dist/lib.js",
-  "types": "./dist/lib.d.ts",
+  "private": false,
+  "description": "Helios is a fast, light Ethereum client for the browser and Node.js.",
+  "keywords": ["ethereum", "light client", "wasm", "rust"],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/evmts/helios.git",
+    "directory": "helios-ts"
+  },
+  "license": "MIT",
+  "contributors": ["Noah Citron <https://x.com/NoahCitron>", "Will Cory <https://x.com/fucory>"],
+  "type": "module",
+  "sideEffects": false,
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": {
+        "import": "./types/esm/lib.d.ts",
+        "require": "./types/cjs/lib.d.cts"
+      },
+      "import": "./dist/lib.js",
+      "require": "./dist/lib.cjs"
+    }
+  },
+  "main": "./dist/lib.cjs",
+  "module": "./dist/lib.js",
+  "types": "./types/esm/lib.d.ts",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack && npm run build:types",
+    "build:types": "npm run build:types:cjs && npm run build:types:esm",
+    "build:types:cjs": "tsc --module CommonJS --moduleResolution node --outDir ./types/cjs && find ./types/cjs -type f -name '*.d.ts' -print0 | xargs -0 -I {} sh -c 'mv \"$1\" \"${1%.d.ts}.d.cts\"' _ {}",
+    "build:types:esm": "tsc --module NodeNext --outDir ./types/esm",
+    "docs": "typedoc",
+    "format": "biome format . --write",
+    "format:check": "biome format .",
+    "lint": "biome check . --write --unsafe",
+    "lint:check": "biome check . --verbose",
+    "lint:package": "bunx publint --strict && attw --pack --ignore-rules cjs-resolves-to-esm",
```

```suggestion
    "lint:package": "npx bunx publint --strict && attw --pack --ignore-rules cjs-resolves-to-esm",
```

Copy pasta this should be npx since this repo isn’t using bun